E4S_Products.md

```diff
@@ -0,0 +1,142 @@
+# E4S Product Dictionary
+
+## E4S Core Packages 
+
+|  Product                                              |   Latest Release        |  Release Date |  Spack Badge | Spack Package | Maintainer | 
+| ----------------------------------------------------- | ----------------------- | --------------|  ----------- | ------------- | ---------- |
+| [adios](https://github.com/ornladios/ADIOS)           | ![release](https://img.shields.io/github/v/release/ornladios/ADIOS) | ![release-date](https://img.shields.io/github/release-date/ornladios/ADIOS) | ![spack](https://img.shields.io/spack/v/adios) | [adios](https://github.com/spack/spack/tree/develop/var/spack/repos/builtin/packages/adios) | 
+| [adios2](https://github.com/ornladios/ADIOS2)         | ![release](https://img.shields.io/github/v/release/ornladios/ADIOS2)  | ![release-date](https://img.shields.io/github/release-date/ornladios/ADIOS2) | ![spack](https://img.shields.io/spack/v/adios2) | [adios2](https://github.com/spack/spack/tree/develop/var/spack/repos/builtin/packages/adios2) |
+| [aml](https://xgitlab.cels.anl.gov/argo/aml)          | | | ![spack](https://img.shields.io/spack/v/aml) | [aml](https://github.com/spack/spack/tree/develop/var/spack/repos/builtin/packages/aml) |
+| [amrex](https://github.com/AMReX-Codes/amrex)         |  ![release](https://img.shields.io/github/v/release/AMReX-Codes/amrex) | ![release-date](https://img.shields.io/github/release-date/AMReX-Codes/amrex) | ![spack](https://img.shields.io/spack/v/amrex) |  [amrex](https://github.com/spack/spack/tree/develop/var/spack/repos/builtin/packages/amrex) |
+| [arborx-x](https://github.com/arborx/ArborX)          | ![release](https://img.shields.io/github/v/release/arborx/ArborX) | ![release-date](https://img.shields.io/github/release-date/arborx/ArborX) | ![spack](https://img.shields.io/spack/v/arborx) | [arborx](https://github.com/spack/spack/tree/develop/var/spack/repos/builtin/packages/arborx) | [aprokop](https://github.com/aprokop)
```

Comment:
minor typo: `arborx-x` should be `arborx`

Reply:
good catch. This is now fixed in https://github.com/E4S-Project/e4s/pull/19/commits/5aed7d88262c6262dd519f2595fcbebf60364306